jina/kubernetes/generic_gateway/gateway.py

```diff
@@ -0,0 +1,14 @@
+import base64
+import sys
+
+from jina import Flow
+
+print('args', sys.argv)
```

Comment:
A gateway should not be handled by a dummy Flow. It should be a Kubernetes deployment where entrypoint is "jina gateway". Btw, jina gateway needs to be offered in the cli/api, will do tomorrow